packageclients/pkg/fakes/crtclient.go

```diff
@@ -0,0 +1,776 @@
+// Code generated by counterfeiter. DO NOT EDIT.
```

Comment:
Question: Where are code gen markers for these fakes located? We will also need a make target within packageclients `Makefile` that generates them.

Reply:
> Question: Where are code gen markers for these fakes located?

It is in the interface definitions for each client

Reply:
> We will also need a make target within packageclients Makefile that generates them.
done.